server/team/team.route.js

```diff
@@ -5,9 +5,20 @@ const { ifNoUserRedirect } = require('../middlewares/user');
 const { ifNoBody400 } = require('../middlewares/util');
 const teamCtrl = require('./team.contoller');
 const validators = require('./team.validation');
+const User = require('../user/user.model');
 
 const router = express.Router(); // eslint-disable-line new-cap
 
+function deleteValidator(req, res, next) {
```

Comment:
This should be in `server/middlewares/team.js` (you will need to create it. Take a look at the user middleware to see how `ifNoUserRedirect` is implemented.